# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/no_libvolume_id.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./extras/volume_id/Makefile.orig	2006-11-25 15:43:05.000000000 -0300
+++ ./extras/volume_id/Makefile	2006-11-25 15:44:03.000000000 -0300
@@ -32,7 +32,7 @@
 	$(Q) $(CC) -c $(CFLAGS) $< -o $@
 
 lib/libvolume_id.a:
-	$(Q) $(MAKE) -C lib
+	$(Q) $(MAKE) -C lib libvolume_id.a
 
 $(PROG): %: $(HEADERS) %.o lib/libvolume_id.a
 	$(E) "  LD      " $@
@@ -56,7 +56,6 @@
 
 install-bin: all
 	$(INSTALL_PROGRAM) -D $(PROG) $(DESTDIR)$(libudevdir)/$(PROG)
-	$(MAKE) -C lib install
 .PHONY: install-bin
 
 uninstall-bin:
